phpBB/styles/prosilver/theme/content.css

```diff
@@ -466,17 +466,31 @@ blockquote cite {
 	/* Username/source of quoter */
 	font-style: normal;
 	font-weight: bold;
-	margin-left: 20px;
+	margin-left: 0;
 	display: block;
 	font-size: 0.9em;
 }
 
+blockquote cite:before {
+	/* Font Awesome quote-left */
+	content: '\f10d';
+	font-family: FontAwesome;
+	padding-right: 5px;
+}
+
 blockquote cite cite {
 	font-size: 1em;
 }
 
 blockquote.uncited {
-	padding-top: 25px;
+	padding-top: 0;
+}
+
+blockquote.uncited:before {
+	/* Font Awesome quote-left */
+	content: '\f10d';
+	font-family: FontAwesome;
+	padding-right: 5px;
 }
```

Comment:
This is also a breaking change if you remove the gifs so keep them for BC until 3.3